linode/manager
# feat: [M3-8017] - Support Linodes in Distributed Compute Regions on Image Create

## Description 📝

- Shows a notice saying `This Linode is in a distributed compute region. Images captured from this Linode will be stored in the closest core site.` if you select a Linode that is in a distributed region on the Image Create page ℹ️ 
- Shows helper text saying `Image will be stored in the closest core site to (us-den-edge-1)` below the Linode select if the selected Linode is in a distributed region ℹ️ 

These changes are based on the Image Service Gen2 Figma wireframes 🎨  - Ask me for a link if you want to reference it.

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-06-04 at 1 53 39 PM](https://github.com/linode/manager/assets/115251059/8096b605-0071-4e4d-b6a6-96fd4ff6d2cb) | ![Screenshot 2024-06-04 at 1 52 41 PM](https://github.com/linode/manager/assets/115251059/8310d7c5-5b98-4c99-b10a-f3b133c72541) |

## How to test 🧪

### Prerequisites
- Have a Linode in a distributed region on your account

### Verification steps
- Go to the Image Create page (http://localhost:3000/images/create/disk)
- Select a Linode that is in a distributed region
- Verify you see the notice and helper text that UX reflects in the mockups 👁️🎨 

## As an Author I have considered 🤔
